firmware/Core/Src/littlefs/flash_driver.c

```diff
@@ -0,0 +1,383 @@
+
+/*-----------------------------INCLUDES-----------------------------*/
+#include "main.h"
+
+#include "littlefs/flash_driver.h"
+#include "debug_tools/debug_uart.h"
+
+// TODO: check on timeout, maybe decrease a lot
+#define FLASH_TIMEOUT_MS 50
+
+// -----------------------------FLASH DRIVER FUNCTIONS-----------------------------
+
+/// @brief Activates the chip select for the given flash module number.
+/// @param chip_number - Index of the chip select line to activate
+/// @details Activates the chip select for the given chip number by setting the corresponding GPIO
+///     pins LOW. Deactivates all other chip selects by setting their corresponding GPIO pins HIGH.
+/// @return None - GPIO writes can't fail
```

Comment:
Is this actually the case lol?

Reply:
Yeah, the HAL GPIO write functions don't have a status return.